backend/src/api/NewsletterBrevo.ts

```diff
@@ -0,0 +1,47 @@
+// eslint-disable-next-line import/no-namespace
+import * as SibApiV3Sdk from '@getbrevo/brevo'
+
+import config from '#config/config'
+import { ContactFormInput } from '#graphql/inputs/ContactFormInput'
+
+export const createBrevoInstance = (): SibApiV3Sdk.TransactionalEmailsApi => {
+  const apiInstance = new SibApiV3Sdk.TransactionalEmailsApi()
+  apiInstance.setApiKey(SibApiV3Sdk.TransactionalEmailsApiApiKeys.apiKey, config.BREVO_KEY)
+  return apiInstance
+}
+
+export const createSmtpEmail = (
+  apiInstance: SibApiV3Sdk.TransactionalEmailsApi,
+  contactFormData: ContactFormInput,
```

Comment:
```suggestion
  contactFormData: ContactForm,
```
This way you have the id, and you can update the db entry when sending the email was successful